chart/README.md

```diff
@@ -541,6 +541,39 @@ Some example commands are shown below.
    sonatype/nexus-iq-server-ha --version <version>
    ```
 
+### Autoscaling
+###### New in version 166.0.0
+IQ HA helm chart includes support for Kubernetes Horizontal Pod Autoscaling (HPA). With this enabled you can set the
+cluster to automatically scale up/down based on cpu and/or memory utilization.
+
+Note: When setting auto-scaling parameters please make sure to have sufficient hardware resources available in the
+underlying nodes meet the max pod demands.
+HPA is disabled by default. If you want to enable it, you need to set the `hpa.enabled` parameter to `true`.
+
+   ```
+   --set hpa.enabled=true
+   ```
+
+HPA requires metrics-server to be installed in the cluster. You can install it by
+running the following
+command: `kubectl apply -f https://github.com/kubernetes-sigs/metrics-server/releases/latest/download/components.yaml`
```

Comment:
with https://github.com/sonatype/nexus-iq-server-ha/pull/17 I think we might be able to modify this (or remove this)

Reply:
Commit: 8fe7a63c8f6e03e20b757197730ca4f625ce777a